src/main/java/com/taskadapter/redmineapi/ProjectManager.java

```diff
@@ -64,7 +64,14 @@
      * 	Project createdProject = mgr.createProject(project);
      * }
      * </pre>
-     *
+     * 
+     * <p>
+     * Note: if {@code project} trackers have not been set with {@link Project#addTrackers}
+     * or cleared with {@link Project#clearTrackers}, 
```

Comment:
I would reword this for clarify to indicate that "or cleared" actually means "or if they have been cleared".

Reply:
Done. Now the or changes to and:

> if trackers have not been set with addTrackers and if they have been cleared with clearTrackers, the created project will get the server default trackers (if any). Otherwise, the project trackers will override the server default settings.

The comment is moved to Project.create() after rebase.